fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -659,9 +675,13 @@ struct LogGenerationData : NonCopyable, public ReferenceCounted<LogGenerationDat
 
 	// only callable after getStorageTeamData returns a null reference
 	Reference<StorageTeamData> createStorageTeamData(StorageTeamID team,
-	                                                 std::vector<Tag>& tags,
-	                                                 std::map<Tag, Version> tagToVersions = {}) {
-		return storageTeamData[team] = makeReference<StorageTeamData>(team, tags, tagToVersions);
+	                                                 const std::vector<Tag>& tags,
+	                                                 bool nothingPersistent,
+	                                                 bool poppedRecently,
+	                                                 bool unpoppedRecovered,
```

Comment:
Can you document each of these parameters?

Reply:
will do

Reply:
actually they are already there in definition, see https://github.com/apple/foundationdb/pull/6327/files#diff-1f725adea23a164d15b5a8d27cfda7e14937da0be7311908af15976b976db9acL483-L486